MAUI/DataForm/layout.md

```diff
@@ -398,4 +398,27 @@ private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs
 {% endhighlight %}
 {% endtabs %}
 
-Here, the `Name` field will be hidden.
\ No newline at end of file
+Here, the `Name` field will be hidden.
+
+## Change the editor height
+
+The data form editor height is changed by using the `EditorHeight` property of the [DataFormItem](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.DataFormItem.html).
+
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
```

Comment:
Remove from upcoming feature
Support for DataFormViewItem’s auto height.

https://help.syncfusion.com/maui/dataform/migration#upcoming-features

Reply:
removed